project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16
```

Comment:
1.0.0?

Reply:
sbt 1.0 is not required to use semanticdb.

Reply:
Fair enough.